src/components/learner-credit-management/tests/BudgetDetailPage.test.jsx

```diff
@@ -756,7 +756,7 @@ describe('<BudgetDetailPage />', () => {
     useBudgetDetailActivityOverview.mockReturnValue({
       isLoading: false,
       data: {
-        contentAssignments: { count: 1 },
+        contentAssignments: { count: 2 },
```

Comment:
[curious] Why the change to `2` here, when the `useBudgetContentAssignments` mock return value below is still only returning 1 result?

Reply:
Oops crust from earlier